src/components/project-tabs/QRTab.jsx

```diff
@@ -1,26 +1,90 @@
-import React, { Component } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import QRCode from "qrcode.react";
+import { FeedbackContext } from "../../contexts/feedback-context";
 import "../../css/project-tabs/QRtab.css";
+import Loader from "react-loader-spinner";
 import "react-bootstrap";
+import moment from 'moment';
+import api from "../../api.js";
+import * as Icon from 'react-bootstrap-icons';
 
-class QRTab extends Component {
-  // eslint-disable-next-line no-useless-constructor
-  constructor(props) {
-    super(props);
+const QRTab = (props) => {
+  // useContext can be used to access global context and dispatch changes
+  const [state, dispatch] = useContext(FeedbackContext);
+
+  useEffect(() => {
+    const param = props.projectID;
+    api
+      .get(`/projects/${param}/conversions`)
+      .then((res) => {
+        if (res.status == 200 && res.data.length > 0) {
+          dispatch({
+            type: "SET_CONVERSIONS",
+            payload: res.data
+          });
```

Comment:
Do you need to check if the status is 200? Wouldn't it just catch the error otherwise it wasn't?

Reply:
Probably not, but it's a good practice, because 200 indicates the response contains the data you're looking for.